bootstrap-app/resources/security/suppression.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't suppose there is a JSON option for configuring the suppression?

Reply:
Sadly I don't think so. The xml schema is so small it really isn't that bad once you get used to it.

https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.2.xsd
https://jeremylong.github.io/DependencyCheck/general/suppression.html

Reply:
https://jeremylong.github.io/DependencyCheck/general/suppression.html

There is at least a button you can click to generate the XML for you from the report.